released/packages/coq-infotheo/coq-infotheo.0.0.5/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "reynald.affeldt@aist.go.jp"
+homepage: "https://github.com/affeldt-aist/infotheo"
+bug-reports: "https://github.com/affeldt-aist/infotheo/issues"
+dev-repo: "git+https://github.com/affeldt-aist/infotheo.git"
+license: "GPLv3"
```

Comment:
```suggestion
license: "GPL-3.0-only"
```
We are trying to switch all projects to [SPDX identifiers](https://spdx.org/licenses/). Unfortunately, "GPLv3" is ambiguous. Can you confirm that this is _only_ GPL 3 and not GPL 3 _or later_? The upstream repo doesn't seem to offer clarity on this.

Reply:
changed to GPL 3 or later